app/controllers/generic_object_definition_controller.rb

```diff
@@ -109,22 +154,22 @@ def custom_button_group_node?
     node.split('-').first == 'cbg'
   end
 
-  def actions_node?
+  def custom_button_node?
     node = x_node || params[:id]
```

Comment:
Can you extract this line outside the methods? I mean: this is new code, so why introduce side channels from the very start.

Make this a function, pass the node in as an agument, extract the first line into `node_info`

Reply:
I have actualy done in myself in https://github.com/ManageIQ/manageiq-ui-classic/pull/2635